scopes/component/isolator/isolator.main.runtime.ts

```diff
@@ -335,12 +342,17 @@ export class IsolatorMain {
     });
     const peerOnlyPolicy = this.getWorkspacePeersOnlyPolicy();
     const capsulesWithModifiedPackageJson = this.getCapsulesWithModifiedPackageJson(capsulesWithPackagesData);
+    if (this.dependencyResolver.hasRootComponents()) {
+      linkingOptions.linkNestedDepsInNM = false;
```

Comment:
let's add a comment about what is this `linkNestedDepsInNM` in the LinkingOptions